Assets/MRTK/Providers/UnityAR/Microsoft.MixedReality.Toolkit.Providers.UnityAR.asmdef

```diff
@@ -2,8 +2,7 @@
     "name": "Microsoft.MixedReality.Toolkit.Providers.UnityAR",
     "references": [
         "Microsoft.MixedReality.Toolkit",
-        "Microsoft.MixedReality.Toolkit.Async",
-        "Unity.XR.ARFoundation"
```

Comment:
Is this change related to the UPM stuff?

Reply:
it is not. must have been accidental. thanks for the catch!